src/pages/404.tsx

```diff
@@ -0,0 +1,43 @@
+import Link from 'next/link';
+import React from 'react';
+
+const Custom404 = () => {
+  return (
+    <div style={styles.container}>
+      <h1 style={styles.title}>404 - Not found</h1>
+      <p style={styles.description}>
+        Something went wrong on our end. Please try again later.
+      </p>
+      <Link href="/">Back home</Link>
+    </div>
+  );
+};
+
+const styles = {
+  container: {
+    display: 'flex',
+    flexDirection: 'column' as 'column',
+    alignItems: 'center',
+    justifyContent: 'center',
+    height: '80vh',
+    textAlign: 'center' as 'center',
```

Comment:
dummy question, why did you did this?  `'center' as 'center'`

Reply:
that was just a TypeScript error, didn't spend much time changing the TS setup as this is a dummy component (we currently don't have a design for this). But it basically is to assert the type of the string to FlexDirection (and TextAlign). using MUI this shouldn't occur